src/coinjoin/client.cpp

```diff
@@ -1879,13 +1876,15 @@ void CCoinJoinClientManager::GetJsonInfo(UniValue& obj) const
     obj.pushKV("sessions",  arrSessions);
 }
 
-void DoCoinJoinMaintenance(CConnman& connman, CBlockPolicyEstimator& fee_estimator, CTxMemPool& mempool)
-{
-    if (coinJoinClientQueueManager != nullptr) {
-        coinJoinClientQueueManager->DoMaintenance();
-    }
+void CJClientManager::Add(CWallet& wallet) {
+    m_imap.emplace(
+        std::make_pair(wallet.GetName(),
```

Comment:
no need to call `make_pair` here:

```suggestion
        m_imap.emplace(wallet.GetName(), std::make_unique....);
```

Reply:
Resolved in latest push